code/lib/types/src/modules/story.ts

```diff
@@ -42,30 +47,40 @@ export type ProjectAnnotations<TRenderer extends Renderer> = CsfProjectAnnotatio
   renderToDOM?: RenderToCanvas<TRenderer>;
 };
 
-export type NormalizedProjectAnnotations<TRenderer extends Renderer = Renderer> =
-  ProjectAnnotations<TRenderer> & {
-    argTypes?: StrictArgTypes;
-    globalTypes?: StrictGlobalTypes;
-  };
+export type NormalizedProjectAnnotations<TRenderer extends Renderer = Renderer> = Omit<
+  ProjectAnnotations<TRenderer>,
+  'decorators' | 'loaders'
```

Comment:
why did you need to omit and reintroduce these types right after?

Reply:
I change the type, the original type also allows non-arrays now.